sdk/storage/azblob/blockblob/client_test.go

```diff
@@ -357,6 +357,36 @@ func (s *BlockBlobUnrecordedTestsSuite) TestBlockBlobClientConnectionString() {
 //		_require.EqualValues(destData, content)
 //	}
 
+func (s *BlockBlobRecordedTestsSuite) TestPutBlobCrcResponseHeader() {
+	_require := require.New(s.T())
+	testName := s.T().Name()
+	svcClient, err := testcommon.GetServiceClient(s.T(), testcommon.TestAccountDefault, nil)
+	_require.NoError(err)
+
+	containerName := testcommon.GenerateContainerName(testName)
+	containerClient := testcommon.CreateNewContainer(context.Background(), _require, containerName, svcClient)
+	defer testcommon.DeleteContainer(context.Background(), _require, containerClient)
+
+	accountName, accountKey := testcommon.GetGenericAccountInfo(testcommon.TestAccountDefault)
+	blobName := testName
+	blobURL := fmt.Sprintf("https://%s.blob.core.windows.net/%s/%s", accountName, containerName, blobName)
+
+	cred, err := blob.NewSharedKeyCredential(accountName, accountKey)
+	_require.NoError(err)
+
+	bbClient, err := blockblob.NewClientWithSharedKeyCredential(blobURL, cred, nil)
+	_require.NoError(err)
+
+	contentSize := 4 * 1024 // 4 KB
+	r, _ := testcommon.GetDataAndReader(testName, contentSize)
+	rsc := streaming.NopCloser(r)
+
+	resp, err := bbClient.Upload(context.Background(), rsc, nil)
+	_require.NoError(err)
+	_require.NotNil(resp)
+	_require.NotNil(resp.ContentCRC64)
```

Comment:
Can you also add a validation step where the crc64 value returned in the response is compared against the one computed for the source data.

Reply:
Added the check